### PR TITLE
labels for not just other instructor, but all

### PR DESCRIPTION
**Pull request by thatrubylove**
maybe I will make it green for "me" and yellow for "other"

### CODE REVIEW

**Comment by joemsak** [comment]
I agree to make "me" green with the correct bootstrap class

**Comment by thatrubylove** [comment]
I went for blue, but it is in

**Comment by thatrubylove** [comment]
also rebased recent master so its a :shipit: as far as I can see

File: app/helpers/one_on_ones_helper.rb [hunk 1]
```diff
@@ -1,10 +1,17 @@
 module OneOnOnesHelper
-  def other_instructor_note(student)
-    unless current_instructor.students.include?(student)
-      content_tag :span,
-                  "Instructor: #{student.instructor_name}",
-                  class: "label label-pill label-warning"
-    end
+  def instructor_note(student)
+    postfix = current_instructor.students.include?(student) ? "mine" : "other"
+    send("instructor_note_#{postfix}", student)
```
Comment (joemsak) [inline app/helpers/one_on_ones_helper.rb]: :+1:

**Pull request merged**